packages/table/src/regions.ts

```diff
@@ -528,6 +528,20 @@ export class Regions {
         return regionGroups;
     }
 
+    public static regionArraysEqual(regionsA: IRegion[], regionsB: IRegion[]) {
+        if (regionsA === undefined && regionsB === undefined) {
+            return true;
+        } else if (regionsA === null && regionsB === null) {
```

Comment:
do you have a good reason for differentiating btw `null` and `undefined` here? given deep equality, those two seem the same to me. 

`null == undefined`